### ISSUE TITLE
Error doing imageMatching using environment

### ISSUE THREAD

**Issue by bgpantojar** [alicevision/meshroom#813]
Hello,

Last weeks I was using just the binaries to get used to meshroom.

Now, I have configured the environment to use the library as a developer. Nevertheless, I have got a problem running one of the basic examples in the ImageMatching node.

![Screenshot from 2020-03-03 12-26-45](https://user-images.githubusercontent.com/35929939/75771741-df16d500-5d4a-11ea-9183-d6ab02b0bdb0.png)

Probably is related with a wrong set up environment because when I lauch the ui I get the next:

![Screenshot from 2020-03-03 12-27-46](https://user-images.githubusercontent.com/35929939/75771864-15eceb00-5d4b-11ea-81b1-6722c52a0c3d.png)

To set up the environment I did basically the next procedure:

1 - download the .zip file from https://github.com/alicevision/meshroom
2 - download the binaries from and copied the folders aliseVision and qtPlugins inside the folder of 1
3 - created the bash files for meshroom.sh and meshroom_ui.sh

![Screenshot from 2020-03-03 13-09-31](https://user-images.githubusercontent.com/35929939/75774433-48e5ad80-5d50-11ea-9f9d-43629abf8926.png)

![Screenshot from 2020-03-03 13-09-45](https://user-images.githubusercontent.com/35929939/75774440-4daa6180-5d50-11ea-8c74-0df77e534c4b.png)

and made them executables

chmod +x meshroom.sh meshroom_ui.sh

This is what the terminal is showing as a error:

![Screenshot from 2020-03-03 14-03-05](https://user-images.githubusercontent.com/35929939/75778313-d7a9f880-5d57-11ea-9310-8e626f8c5f59.png)

![Screenshot from 2020-03-03 14-03-14](https://user-images.githubusercontent.com/35929939/75778326-dbd61600-5d57-11ea-9321-56e6b4d43fb5.png)

What could be the problem?

**Comment by gryan315** [comment]
Well, you could fix the "unrecognized option --method" by removing the method argument from the imagematching python file under lib/meshroom/nodes/alicevision. It will just always use the SIFT tree, which is how the current build works. I had to remove a lot of added UI choices from the files when I was testing the dev build with the live alicevision binaries.

**Comment by bgpantojar** [comment]
Hello.. I tried doing that and did not work. Now the ui is even not opening when is called.

![Screenshot from 2020-03-10 10-16-13](https://user-images.githubusercontent.com/35929939/76297517-3b807400-62b8-11ea-8399-a06328bc7cfb.png)

**Comment by fabiencastan** [comment]
Hi,
In python `'''` is not a comment. It creates a string.
Usually you can have a useless string in the middle of the code. But here you are in the middle of a list of elements, so you cannot do that.
You should have the error message explaining the problem at the very beginning of your shell where you launched Meshroom.

But Meshroom is intended to be used with a specific version of AliceVision. Trying to mix versions can be complex as it is not done for that. And I'm not sure what you will gain from updating just the UI.

**Comment by bgpantojar** [comment]
Hello,

I am needing the dense point cloud as .ply file and calculate the points normal information. For the first, according to a past issue, I know that is available in the source code but not in the last release. For the second, I was planing to implement a code in meshroom files to get that information. In that way, I was trying to build the developer version of the program. I tried to build aliceVision, but a lot of issues were found. So, I tried to follow the instructions of this issue ( https://github.com/alicevision/meshroom/issues/690), to reuse  the aliceVision and qtPlugins folders from the last release binaries.
